nixos/modules/config/stevenblack.nix

```diff
@@ -1,34 +1,51 @@
-{ config, lib, pkgs, ... }:
-
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
 let
-  inherit (lib) optionals mkOption mkEnableOption types mkIf elem concatStringsSep maintainers;
-  cfg = config.networking.stevenblack;
+  inherit (lib)
+    elem
+    getOutput
+    maintainers
+    mkEnableOption
+    mkIf
+    mkPackageOption
+    mkOption
+    optionals
+    types
+    ;
```

Comment:
```suggestion
  inherit (lib)
    getOutput
    maintainers
    mkEnableOption
    mkIf
    mkPackageOption
    mkOption
    types
    ;
```

Reply:
Oh god, the diff looks ugly...

Remove `elem` and `optionals`

Reply:
Also sorting them (via Vim's `:sort`) is recommended

Reply:
It does look kinda ugly 😭 but this is how nixfmt wanted it. Will sort via nvim sort tho